### ISSUE TITLE
The "Forward" button in no longer active after navigating back to homepage if a website was opened while the content feed preview was opened

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2268]
**[Affected Versions]:**
- Firefox Reality version 6 RC5 (33a031c7)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened.

**[Steps to reproduce]:**
1. Click on a content feed from homepage.
2. Navigate to a website using the Awesomebar.
2. Click the "Back" button 2 times so that the homepage is displayed.
3. Observe the "Forward" button.

**[Expected results]:**
- The button is active.

**[Actual results]:**
- The button is NOT active.

**[Notes]:**
- This issue is also reproducible in the production version of the app (v5).
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1IRDDw0Nh52ONNmnt0EWpFx6Jyei4pRay).

**Comment by keianhzo** [comment]
This seems to be the https://webxr.today page altering the history, happens in any browser.

**Comment by KVoynar** [comment]
Yes, confirmed. I tested this on the Quest on multiple thumbnails and duplicated this result every time.

**Comment by keianhzo** [comment]
Related to https://github.com/MozillaReality/FirefoxReality/issues/2266

**Comment by jvonitter** [comment]
@joshmarinacci i think this one should be over in your repo
